wire-ios-request-strategy/Sources/Payloads/Payload.swift

```diff
@@ -47,11 +47,35 @@ enum Payload {
         var qualifiedIDs: [QualifiedID]
     }
 
+    struct UserProfilesList: Codable {
+
+        enum CodingKeys: String, CodingKey {
+            case found = "found"
+            case failed = "failed"
+        }
+
+        let found: [Payload.UserProfile]
+        let failed: [QualifiedID]?
+
+    }
+
     struct Prekey: Codable {
         let key: String
         let id: Int?
     }
 
+    struct PrekeyByQualifiedUserIDWithFailedUsers: Codable {
```

Comment:
Similar comment here, although it's clear enough by the name what this struct represents, maybe renaming it to `PrekeyByQualifiedUserIDV4` would be good for consistency (if we also rename `UserProfilesList`)

Reply:
Done